src/Components/Components/src/RenderTree/RenderTreeDiffBuilder.cs

```diff
@@ -18,6 +18,8 @@ enum DiffAction { Match, Insert, Delete }
     // hit the "old < new" code path during diffing so we only have to check for it in one place.
     public const int SystemAddedAttributeSequenceNumber = int.MinValue;
 
+    private static RenderTreeFrame NoSuchAttributeFrame = RenderTreeFrame.Attribute(int.MaxValue, null, null);
```

Comment:
Good point - have made it readonly now.

Reply:
Actually I've changed it further to avoid having `NoSuchAttributeFrame` at all. If two of us on the team can't immediately tell whether that was going to cause struct copying, it's not a language feature we should be relying on.